diff --git a/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js b/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
index 79c2168..5d5767e 100644
--- a/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
+++ b/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
@@ -84,7 +84,7 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
 
   const menuTitle = formatMessage({
     id: 'app.components.LeftMenu.navbrand.title',
-    defaultMessage: 'Strapi Dashboard',
+    defaultMessage: 'Dashboard',
   });
 
   return (
diff --git a/node_modules/@strapi/admin/admin/src/pages/Admin/index.js b/node_modules/@strapi/admin/admin/src/pages/Admin/index.js
index 6d23312..dcd4dcd 100644
--- a/node_modules/@strapi/admin/admin/src/pages/Admin/index.js
+++ b/node_modules/@strapi/admin/admin/src/pages/Admin/index.js
@@ -88,7 +88,7 @@ const Admin = () => {
       >
         <Suspense fallback={<LoadingIndicatorPage />}>
           <Switch>
-            <Route path="/" component={HomePage} exact />
+            <Route path="/" component={CM} exact />
             <Route path="/me" component={ProfilePage} exact />
             <Route path="/content-manager" component={CM} />
             {routes}
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
index f8b8d85..28a56d8 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
@@ -66,7 +66,7 @@ const HomePage = () => {
         <LogoContainer>
           <img alt="" aria-hidden src={Logo} />
         </LogoContainer>
-        <Box padding={10}>
+        {/* <Box padding={10}>
           <Grid>
             <GridItem col={8} s={12}>
               <HomeHeader
@@ -83,7 +83,7 @@ const HomePage = () => {
               <SocialLinks />
             </GridItem>
           </Grid>
-        </Box>
+        </Box> */}
       </Main>
     </Layout>
   );
